packages/tests/src.ts/test-utils.spec.ts

```diff
@@ -441,20 +441,23 @@ xdescribe("Test Signature Manipulation", function() {
             const sig = hethers.utils.splitSignature({ r: r, s: s, v: v });
 
             {
+                // @ts-ignore
                 const addr = hethers.utils.recoverAddress(hash, {
                     r: r, s: s, v: v
                 });
                 assert.equal(addr, address, "Using r, s and v");
             }
 
             {
+                // @ts-ignore
                 const addr = hethers.utils.recoverAddress(hash, {
                     r: sig.r, _vs: sig._vs
                 });
                 assert.equal(addr, address, "Using r, _vs");
             }
 
             {
+                // @ts-ignore
                 const addr = hethers.utils.recoverAddress(hash, {
                     r: sig.r, s: sig.s, recoveryParam: sig.recoveryParam
                 });
```

Comment:
Shouldn't we remove these lines if `recoverAddress` is no longer present?

Reply:
I removed the whole test as it was already disabled.